control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -981,6 +981,7 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
         request: tonic::Request<ControllerExpandVolumeRequest>,
     ) -> Result<tonic::Response<ControllerExpandVolumeResponse>, tonic::Status> {
         let request = request.into_inner();
+        info!("Volume expansion request: {:?}", &request);
```

Comment:
in line with existing methods:
```suggestion
        tracing::trace!(volume.uuid = %args.volume_id, request = ?args);
```